doc/examples/reconst_dki.py

```diff
@@ -48,99 +53,99 @@
                     & W_{xxzz} & W_{yyzz} & W_{xxyz} & W_{xyyz} & W_{xyzz}
                     & & )\end{matrix}
 
-In the following example we show how to reconstruct your diffusion multi-shell
-datasets using the kurtosis tensor model.
-
-First import Dipy's DKI module:
+In the following example we show how to fit the diffusion kurtosis model on
+diffusion-weighted multi-shell datasets and how to estimate diffusion kurtosis
+based statistics.
 
+First, we import all relevant modules:
 """
 
 import dipy.reconst.dki as dki
+import dipy.reconst.dti as dti
+import matplotlib.pyplot as plt
+import nibabel as nib
+from dipy.data import fetch_cenir_multib
+from dipy.data import read_cenir_multib
+from dipy.segment.mask import median_otsu
+from dipy.denoise.noise_estimate import estimate_sigma
+from dipy.denoise.nlmeans import nlmeans
 
 """
 DKI requires multi-shell data, i.e. data acquired from more than one non-zero
-b-value. Here, we use fetch to download the raw HARDI of a dMRI dataset
-cotaining three non-zero b-values. The size of the dataset is 188 MBytes,
-however you only need to fetch it once.
+b-value. Here, we use fetch to download a multi-shell dataset with parameters
+that are similar to the data collected in the Human Connectome Project (HCP).
+The total size of the downloaded data is 1760 MBytes, however you only need to
+fetch it once. Parameter ``with_raw`` of function ``fetch_cenir_multib`` is set
+to ``False`` to only download eddy-current/motion corrected data:
 """
 
-from dipy.data import fetch_sherbrooke_3shell
-
-fetch_sherbrooke_3shell()
+fetch_cenir_multib(with_raw=False)
 
 """
-Next, we read the saved dataset
+Next, we read the saved dataset. To decrease the influence of diffusion signal
+Taylor approximation components larger than the fourth order (componets not
+taken into account by the diffusion kurtosis tensor), we only select the
+b-values up to 2000 $s.mm^{-2}$:
 """
 
-from dipy.data import read_sherbrooke_3shell
+bvals = [200, 400, 1000, 2000]
 
-img, gtab = read_sherbrooke_3shell()
+img, gtab = read_cenir_multib(bvals)
```

Comment:
Does this actually work for you? It seems that read_cenir_multib returns (gtab, img) and not the other way around